src/utils/deployRollup.ts

```diff
@@ -205,9 +226,14 @@ export async function deployRollup({
     'deployed-at': rollupContracts.deployedAtBlockNumber,
   };
 
-  l3Config.rollup= rollupCreatedEvent.args.rollupAddress;
+  rollupConfigData.node['batch-poster']['parent-chain-wallet']['private-key'] =
```

Comment:
Let's make sure the `l3Config` is also updated: https://github.com/OffchainLabs/orbit-deployment-ui/blob/main/src/components/SetBatchPoster.tsx#L42